src/howtoplay/howtoplay.scss

```diff
@@ -0,0 +1,64 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap");
```

Comment:
If you use `.scss` why do you need `.css` files 😄 ?

Reply:
You can create a JS file called `howToPlay.js` for example and you can import in it `./howtoplay.scss`. After building the version with Webpack - it should solve the problem with styling itself. For tests, it is good to create a 'scss' file and use the loader to translate it to 'css' and check if the page looks good - but there is no need to push the CSS file later.

Reply:
Ohh and name files using `camelCase` notation.

Reply:
Imported styles in howtoplay.js